e2e/tests/hmctsAdministersCaseAfterSubmission_test.js

```diff
@@ -0,0 +1,84 @@
+const config = require('../config.js');
+
+let caseId;
+
+Feature('Case administration after submission');
+
+Before(async (I, caseViewPage, submitApplicationEventPage) => {
+  if (!caseId) {
+    await I.logInAndCreateCase(config.swanseaLocalAuthorityEmailUserOne, config.localAuthorityPassword);
+    await I.enterMandatoryFields();
+    caseViewPage.goToNewActions(config.applicationActions.submitCase);
+    submitApplicationEventPage.giveConsent();
+    I.continueAndSubmit();
+
+    // eslint-disable-next-line require-atomic-updates
+    caseId = await I.grabTextFrom('.heading-h1');
+    console.log(`Case ${caseId} has been submitted`);
+
+    I.wait(2); // in seconds; time needed for access grant calls to complete
```

Comment:
Do we need to wait for access calls here? We are not testing the access segregation here. If we don't does it make this test flaky?

Reply:
Logically seems not needed, I might have left it by mistake. Will remove and report back.

It should not have any negative impact, just 2 more secs to exec time.